src/main/java/me/zeroeightsix/kami/module/modules/player/Scaffold.java

```diff
@@ -35,13 +37,16 @@
 public class Scaffold extends Module {
 
     private Setting<Boolean> placeBlocks = register(Settings.b("Place Blocks", true));
+    private Setting<Boolean> tower = register(Settings.b("Tower", false));
     private Setting<Mode> modeSetting = register(Settings.enumBuilder(Mode.class).withName("Mode").withValue(Mode.LEGIT).build());
     private Setting<Boolean> randomDelay = register(Settings.booleanBuilder("Random Delay").withValue(false).withVisibility(v -> modeSetting.getValue().equals(Mode.LEGIT)).build());
     private Setting<Integer> delayRange = register(Settings.integerBuilder("Delay Range").withMinimum(0).withValue(6).withMaximum(10).withVisibility(v -> modeSetting.getValue().equals(Mode.LEGIT) && randomDelay.getValue()).build());
     private Setting<Integer> ticks = register(Settings.integerBuilder("Ticks").withMinimum(0).withMaximum(60).withValue(2).withVisibility(v -> !modeSetting.getValue().equals(Mode.LEGIT)).build());
 
     private boolean shouldSlow = false;
 
```

Comment:
This doesn't need a second empty line between, because you're only declaring 2 related variables